packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/datebox.mask.tests.js

```diff
@@ -1243,6 +1226,51 @@ module('Search', setupModule, () => {
     });
 });
 
+module('Date AM/PM Handling', {
+    beforeEach: function() {
```

Comment:
if we have nothing to do additionally in `beforeEach` we can just pass already existing setupModule